draft-packages/avatar/KaizenDraft/Avatar/Avatar.tsx

```diff
@@ -28,6 +28,11 @@ export interface AvatarProps {
    * We use this for the alt text of the avatar, and to derive intials when user has no avatar image.
    */
   fullName: string
+  /**
+   * Default behaviour when an avatarSrc is not provided is to generate initials from the username.
+   * This disables this feature and shows the generic avatar.
+   */
+  disableInitials?: string
```

Comment:
/nitpick I think we can do a little more to inform when the user should use this option. Here's an excerpt from the docs - "When there is no specific individual or group identified, use the default user avatar."

Reply:
I've updated the description to include this